src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const costPerDay = 40;
+  let totalCost = days * costPerDay;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Instead of using a hardcoded value for the number of days and the discount, define constants with descriptive names to represent these values. This will make the code more readable and maintainable. Refer to the checklist item 2 for the correct example.